src/main/java/com/crowdin/client/labels/LabelsApi.java

```diff
@@ -141,5 +144,32 @@ public ResponseList<SourceString> unassignLabelFromStrings(Long projectId, Long
         return SourceStringResponseList.to(response);
     }
 
+    /**
+     * @param projectId Project Identifier
+     * @param labelId Label Identifier
+     * @param request Request Object
+     * @return Screenshots
+     * @see <ul>
+     * <li><a href="https://developer.crowdin.com/api/v2/#operation/api.projects.labels.screenshots.post" target="_blank"><b>API Documentation</b></a></li>
+     * </ul>
+     */
+    public ResponseList<Screenshot> assignLabelToScreenshots(Long projectId, Long labelId, LabelToScreenshotsRequest request) throws HttpException, HttpBadRequestException{
+        String builtUrl = String.format("%s/projects/%d/labels/%d/screenshots", this.url, projectId, labelId);
+        ScreenshotResponseList response = this.httpClient.post(builtUrl, request, new HttpRequestConfig(), ScreenshotResponseList.class);
+        return ScreenshotResponseList.to(response);
+    }
 
+    /**
+     * @param projectId Project Identifier
+     * @param labelId Label Identifier
+     * @return Screenshots
+     * @see <ul>
+     * <li><a href="https://developer.crowdin.com/api/v2/#operation/api.projects.labels.screenshots.deleteMany" target="_blank"><b>API Documentation</b></a></li>
+     * </ul>
+     */
+    public ResponseList<Screenshot> unassignLabelFromScreenshots(Long projectId, Long labelId) throws  HttpException, HttpBadRequestException {
```

Comment:
the required query parameter `screenshotIds` is missing here

Reply:
Sorry, Not sure how to incorporate that. Checked the unassignLabelFromStrings API but there was nothing related to this. Please suggest how can i do that.

Reply:
@Durdush just add the third parameter `screenshotIds` to the `unassignLabelFromScreenshots`, build query params, and pass to the `httpClient.delete` like in any get method by analogy

Reply:
by the way, the `unassignLabelFromStrings` method is outdated, it would be great to also fix this according to the docs

Reply:
If i add the screenshotIds as query param in httpClient.delete then the URL will change.
Current: {protocol}://{host}/api/v2/projects/{projectId}/labels/{labelId}/screenshots
Then: {protocol}://{host}/api/v2/projects/{projectId}/labels/{labelId}/screenshots/{screenshotId}
is it alright? Also for multiple should i loop though each of the screenshotId. Please suggest good way if any.

Reply:
And can i do the similar changes in unassignLabelFromStrings API in the same issue ?